ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Client.scala

```diff
@@ -331,8 +331,15 @@ private[ember] class H2Client[F[_]](
 }
 
 private[ember] object H2Client {
+  private[this] type SwitchBoard[F[_]] = Map[H2Client.RequestKey, (H2Connection[F], F[Unit])]
+
+  implicit private[this] class SwitchBoardOps[F[_]](private val sb: SwitchBoard[F]) extends AnyVal {
```

Comment:
It's definitely bikeshedding, but perhaps, `AnyVal` won't do any magic because of parametrization, won't it?